src/main/java/org/jitsi/videobridge/cc/BandwidthProbing.java

```diff
@@ -32,16 +34,45 @@
     extends PeriodicRunnable
 {
     /**
+     * The system property name that holds a boolean that determines whether or
+     * not to activate the RTX bandwidth probing mechanism that implements
+     * stream protection.
+     */
+    public static final String
+        DISABLE_RTX_PROBING_PNAME = "org.jitsi.videobridge.DISABLE_RTX_PROBING";
```

Comment:
We can set the default to whatever we want, but conceptually in general the default value of a boolean is false. I don't mind to changing this but we try to follow this rule across the projects.